shell/platform/darwin/ios/framework/Source/FlutterOverlayView.mm

```diff
@@ -0,0 +1,76 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/darwin/ios/framework/Source/FlutterOverlayView.h"
+
+#include "flutter/common/settings.h"
+#include "flutter/common/task_runners.h"
+#include "flutter/flow/layers/layer_tree.h"
+#include "flutter/fml/platform/darwin/cf_utils.h"
+#include "flutter/fml/synchronization/waitable_event.h"
+#include "flutter/fml/trace_event.h"
+#include "flutter/shell/common/platform_view.h"
+#include "flutter/shell/common/rasterizer.h"
+#include "flutter/shell/platform/darwin/ios/ios_surface_gl.h"
+#include "flutter/shell/platform/darwin/ios/ios_surface_software.h"
+#include "third_party/skia/include/utils/mac/SkCGUtils.h"
+
+@implementation FlutterOverlayView
```

Comment:
As written, there seems to be no functional difference between a FlutterView and FlutterOverlayView. Can they me merged with the few differences between the two accounted for by patching the original FlutterView?

Reply:
TODO is fine.

Reply:
Discussed offline, added a TODO to revisit this and see if we can merge after we land GL support.